### DISCUSSION TITLE
Error importing local CSS modules in v9.5.0

### DISCUSSION THREAD

**Discussion by justincy** [vercel/next.js#15540] [Help] [answered]
After upgrading to 9.5.0, I'm getting the following build error:

> ./src/components/Header/Header.module.css
> CSS Modules cannot be imported from within node_modules.
> Read more: https://err.sh/next.js/css-modules-npm
> Location: src/components/Header/Header.tsx

I don't quite understand what's wrong with what I'm doing. This is a local CSS module import. It isn't an import inside of node_modules. The import line looks like this:

```js
import styles from './Header.module.css';
```

Is it not correctly detecting that `./src/components/Header/Header.module.css` is inside my local project directory?

**Comment by jamesmosier** [comment]
Hi @justincy. Do you happen to have a repository with a sample of what you are seeing? I wasn't able to recreate this 🤔 

Also I've had trouble in the past with some upgrades, so it might be worth deleting your `.next` and `node_modules` folders and reinstalling/rebuilding to see if that helps. Not a great solution, but my only shot in the dark guess 😄 

<img width="1084" alt="Screen Shot 2020-07-27 at 8 53 00 PM" src="https://user-images.githubusercontent.com/2854919/88606640-35aee380-d04b-11ea-9c30-6576d48fbcd6.png">

**Reply by justincy** [reply]
Thanks for looking at this. It looks like my issue is related to next-transpile-modules; they're having a little trouble with the upgrade to webpack 5 (https://github.com/martpie/next-transpile-modules/pull/96).

**Comment by justincy** [comment, marked as answer]
Turns out my problem was cause by a bug in next-transpile-modules v4.0.0. Upgrading to v4.0.1 (which was just released) fixed my issue.
